### PR TITLE
[electroluxair] Remove binding

### PR DESCRIPTION
**Pull request by lsiepel**
Because https://github.com/openhab/openhab-addons/pull/17324 caused conflicts when backporting i manually created this PR against 4.2

### CODE REVIEW

File: bundles/effective-pom.xml
Comment (jlaur) [inline bundles/effective-pom.xml:1]: This looks like it should not have been included?

Comment (lsiepel) [reply bundles/effective-pom.xml:1]: Nice spot. This file keeps coming back, it slipped trough.

**Comment by lsiepel** [comment]
Don't know why it complains about spotless, the lines it is complaining bout are not changed by this PR.

**Review by wborn — APPROVED** [review]
Thank you!

**Pull request merged**